src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -353,6 +354,11 @@ struct SvgNode
     ~SvgNode();
 };
 
+struct SvgCompositeNode : SvgNode
+{
+    bool objectBBox = false;
```

Comment:
@JSUYA for the gradient `objectBoundingBox` is the default units value. so `userSpace == true` only on request.
For clip and mask, the default is `userSpaceOnUse`, so by the analogy, I decided to use a different name, so that `objectBBox == true` only on request.

do you think it's less clear? if so I can change it

Reply:
if you give the meaning of default value, you are right.
I just thought that it would be better for if the unit of gradient and unit of clip-path were declared with similar names.
If someday we need to support pattern and support patternUnit, its name will be objectBBbox.
+
We already use bbox in engine code, but we decided not to use abbreviations, so I don't know if it's better to keep BBox. hahahaha....

Reply:
changed